src/components/Modals/index.tsx

```diff
@@ -0,0 +1,670 @@
+          mt={'24px'}
+          padding={'22px 24px'}
+          borderRadius={'10px'}
+          border={'solid 1px #282d3d'}
+          display={'flex'}
+          justifyContent={'space-between'}
+          alignItems={'center'}
+        >
+          <Box>Enable as collateral</Box>
+          <AntSwitch
+            inputProps={{ 'aria-label': 'ant design' }}
+            defaultChecked={modalSetting.setModalIsConfirm.value}
+          />
+        </Box>
+      )}
+      <Box mt={'24px'} display={'flex'}>
+        {value > 0 ? (
+          <MuiButton
+            fullWidth
+            style={{
+              cursor: 'pointer',
+            }}
+            // sx={{
+            //   color: '#696c80',
+            //   borderRadius: '10px',
+            //   bgcolor: '#3e4252',
+            //   textAlign: 'center',
+            //   fontSize: '16px',
+            //   fontWeight: '600',
+            // }}
+            onClick={() => {
+              modalSetting.setModalType.set('state');
+              modalSetting.setModalNotoolbar.set(true);
+            }}
+          >
+            Confirm
+            {/* {!modalSetting.setModalIsConfirm.value ? 'Enter amount' : 'Confirm'} */}
+          </MuiButton>
+        ) : (
+          <Box
+            width={'100%'}
+            py={'16px'}
+            borderRadius={'10px'}
+            bgcolor={'#3e4252'}
+            textAlign={'center'}
+            fontSize={'16px'}
+            fontWeight={'600'}
+            color={'#696c80'}
+          >
+            Enter amount
+          </Box>
+        )}
+      </Box>
+    </Box>
+  );
+};
+interface StateModalContentProps extends ModalContentProps {
+  loading?: boolean;
+  setOpenModalType?: any;
+}
+export const StateModalContent: React.FC<StateModalContentProps> = ({
+  modalSetting,
+  loading,
+  setOpenModalType,
+}) => {
+  const [isLoading, setIsLoading] = useState(loading ? true : false);
+  useEffect(() => {
+    window.setTimeout(() => {
+      isLoading && setIsLoading(false);
+    }, 3000);
+  }, [isLoading]);
+  return (
+    <>
+      {isLoading ? (
+        <Box position={'relative'} width={'369px'}>
+          <img
+            src={
+              require('../../assets/images/resource/loadingmodalback.svg')
+                .default
+            }
+          />
+          <Box
+            position={'absolute'}
+            top={'0px'}
+            left={'0px'}
+            width={'100%'}
+            height={'100%'}
+            display={'flex'}
+            flexDirection={'column'}
+            alignItems={'center'}
+            justifyContent={'center'}
+            gridGap={'32px'}
+          >
+            <Spinner>
+              <img
+                src={
+                  require('../../assets/images/resource/spinner.svg').default
+                }
+              />
+            </Spinner>
+            <Box fontSize={'18px'} fontWeight={'500'} color={'#c7cad9'}>
+              Confirm transaction in your wallet

```

Comment:
Messages should come from the messages file talk to murai if you don't know where